src/dmadata/dmadata.c

```diff
@@ -1,7 +1,7 @@
 #include "z64dma.h"
 
 // Linker symbol declarations (used in the table below)
-#define DEFINE_DMA_ENTRY(name)            \
+#define DEFINE_DMA_ENTRY(name, _1)        \
```

Comment:
I considered this but decided against it because it would look confusing inside the `_##segmentName##SegmentRom` stuff which are fairly hard to read even with distinct words. It's also the sort of macro you wouldn't touch unless you knew what kind of name we're talking about so I felt it could remain as just `name`. What do you think?

Reply:
I agree it's not a big deal so if you prefer `name` fine by me